src/main/kotlin/org/opensearch/commons/alerting/model/NoOpTrigger.kt

```diff
@@ -51,19 +54,17 @@ data class NoOpTrigger(
 
         @JvmStatic @Throws(IOException::class)
         fun parseInner(xcp: XContentParser): NoOpTrigger {
-            if (xcp.currentToken() != XContentParser.Token.START_OBJECT && xcp.currentToken() != XContentParser.Token.FIELD_NAME) {
-                XContentParserUtils.throwUnknownToken(xcp.currentToken(), xcp.tokenLocation)
-            }
-
-            // If the parser began on START_OBJECT, move to the next token so that the while loop enters on
-            // the fieldName (or END_OBJECT if it's empty).
+            var id = ""
```

Comment:
Lets generate an id in case we want to have this used by monitors. Something llike:
```
var id = UUIDs.base64UUID()
```